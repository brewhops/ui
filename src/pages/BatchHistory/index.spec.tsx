import React from 'react';
import { BatchHistory } from '.';
import { shallow } from 'enzyme';

describe('src/pages/BatchHistory/index.tsx', () => {
  it('renders the Batch History page', () => {
    expect(shallow(<BatchHistory/>)).toMatchSnapshot();
  })
});