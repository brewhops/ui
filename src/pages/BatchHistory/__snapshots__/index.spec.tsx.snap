// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/pages/Admin/NavBar.tsx renders the nav bar 1`] = `ShallowWrapper {}`;

exports[`src/pages/BatchHistory/index.tsx renders the Batch History page 1`] = `ShallowWrapper {}`;
